### PR TITLE
Update devexpress-web package: to versions: 16.1.12 (24-MAY-2017), 16.2.7 (24-MAY-2017), 17.1.3 (17-MAY-2017).

### PR DESCRIPTION
**Pull request by romanresh**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
https://documentation.devexpress.com/#AspNet/DevExpressWebScripts
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by mhegazy** [comment]
@romanresh  only major versions are supported in folders names. that is what is causing the build failure.

**Comment by romanresh** [comment]
Hi @mhegazy . I believe I've found the cause of this problem. This definitions package is a part of the DevExpress ASP.NET components library, but we use the different version system (non semver) in our product. I've changed the folders names and definitions headers to make them compatible with semver.

File: types/devexpress-web/README.md [hunk 1]
```diff
@@ -2,8 +2,8 @@ DevExpress ASP.NET/MVC TypeScript definitions
 =============================================
 
 You can use these TypeScript definitions for your web projects, which contain
-DevExpress ASP.NET or MVC controls. For this, simply add  at the top of your
-code `<reference path="TypeScript/devexpress-web.d.ts" />`.
+DevExpress ASP.NET or MVC controls. For this, simply add at the top of your
+code `<reference path="index.d.ts" />`.
```
Comment (andy-ms) [inline types/devexpress-web/README.md]: That reference won't work unless you're in the same directory as `index.d.ts`. You probably meant `/// <reference types="devexpress-web" />`.
We don't publish custom READMEs, so anything important should be moved to a comment in `index.d.ts` instead.

Comment (romanresh) [inline types/devexpress-web/README.md]: Hi @andy-ms. I've removed mention of the reference path from the README file. We plan to update the article about using TS definition in our documentation. With the new version of DevExpress controls (and their TS definition resp.) I'll add a link to this article.

**Comment by romanresh** [comment]
Hi @mhegazy @andy-ms . Any new with this PR? Is it OK now, or should I fix anything else?

**Comment by andy-ms** [comment]
Looks good.

**Pull request merged**